src/Handler/TagHandler.php

```diff
@@ -80,7 +86,14 @@ static function ($tag) use ($repoPrefix) {
                 $tags[] = 'ez-all';
             }
 
-            $response->headers->set($this->tagsHeader, implode(' ', array_unique($tags)));
+            $tagsString = implode(' ', $tags);
+            if ($this->tagsHeaderMaxLength && strlen($tagsString) > $this->tagsHeaderMaxLength) {
+                $tagsString = trim(substr($tagsString, 0, strrpos(
```

Comment:
I'd like us to add a warning here when we end up dropping tags.

Reply:
Done, warning message:
HTTP Cache tags header max length reached and truncated to `$this->tagsHeaderMaxLength`
Let me know if it should be updated.